wagtail_localize_rws_languagecloud/emails.py

```diff
@@ -13,22 +14,72 @@ def get_full_url(url):
     return url
 
 
-def send_emails(translation):
-    subject, body = compose_email(translation)
+# sync_rws command emails
+def send_sync_rws_emails(translation):
+    subject, body = compose_sync_rws_email(translation)
     recipients = get_recipients()
     for recipient in recipients:
         send_mail(subject, body, [recipient])
 
 
-def compose_email(translation):
+def compose_sync_rws_email(translation):
     subject = gettext_lazy("Translated content ready for review")
     body = gettext_lazy(
         "Translated content for '%(instance)s' is ready for review at: %(edit_url)s"
-        % {
-            "instance": str(translation.get_target_instance()),
-            "edit_url": get_full_url(translation.get_target_instance_edit_url()),
+    ) % {
+        "instance": str(translation.get_target_instance()),
+        "edit_url": get_full_url(translation.get_target_instance_edit_url()),
+    }
```

Comment:
Found a bug here where interpolation was done on the plain string instead of the translated string.